internal/tofu/context_functions_test.go

```diff
@@ -635,18 +635,126 @@ variable "obfmod" {
 		},
 	})
 
-	diags := ctx.Validate(m)
+	_, diags := ctx.Plan(m, nil, nil)
```

Comment:
It seems like some other work is still bleeding into here somehow. The changes in this file seem to be adjacent to https://github.com/opentofu/opentofu/pull/2082, right? :thinking:

Reply:
Correct, though this particular scenario now fails clearly in plan instead of validate now.